Makefile.one

```diff
@@ -0,0 +1,468 @@
+.PHONY: all
+all:
+	$(foreach var,$(.VARIABLES),$(info $(var) = $($(var))))
+
+	#$(MAKE) env
+	#$(MAKE) reqs
+	#$(MAKE) tracee-ebpf tracee-rules rules
```

Comment:
So, all (the default target?) doesn't include the bpf-noncore target anymore, right?
Although this is good for environments with BTF enabled, it means that on environments without BTF the user will need to build the bpf object sepaeately.
This is ok, as long as we understand that, but don't we want to inform the user in such cases?

Reply:
I think it's a good call, given us de-emphasizing non-core. if someone wants to build the bpf target, they will probably do not want to build all of tracee in the same place.

Reply:
I can document it indeed. If you see the docker makefile, Ive made sure to have it working for non-core by including the kernel headers bind mounts (and I tested it with non BTF environments).

 I think I should mention this in building documentation indeed.

Reply:
https://github.com/aquasecurity/tracee/issues/1233#issuecomment-992487235